src/Timing/tdis.f90

```diff
@@ -314,7 +289,7 @@ subroutine tdis_set_delt()
       delt = tsmult(kper) * delt
     end if
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine tdis_set_delt
 
```

Comment:
There appears to be an commented-out function ('tdis_set_delt_std') in tdis.f90 that I can remove as part o this PR, if you like?  The original line numbers look to be 321-373.  The new line numbers under this PR equate to 296-348